README.md

```diff
@@ -8,12 +8,9 @@ This can be especially useful for publishing static website, such as with
 [GitHub Pages](https://pages.github.com/), from built files in other job
 steps, such as [Doxygen](http://www.doxygen.nl/) generated HTML files.
 
-**NOTE**: GitHub currently requires the use of a Personal Access Token for
-pushing to other repositories. Pushing to the current repository should work
-with the always-available GitHub Token (available via
-`{{ secrets.GITHUB_TOKEN }}`. If pushing to another repository, a Personal
-Access Token will need to be [created](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) and assigned to the
-workflow [secrets](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables).
+**NOTE**: You'll need a Personal Access Token if you want to push to another
+repo or have Two-factor authentication set up. You'll need to [create](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) a
```

Comment:
Hmm, interesting. I have two-factor setup for my account and don't seem to need this. Do you mean that the repository/org has mandatory two-factor authentication for contributors? (In which case the docs should say that more explicitly, if so.)

Reply:
Actually it seems like the `GITHUB_TOKEN` gives standard permissions listed [on GitHub's documentation](https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token#permissions-for-the-github_token).

It probably broke for me because the branch didn't exist first and the `GITHUB_TOKEN` doesn't give access to "[Administration](https://developer.github.com/v3/apps/permissions/#permission-on-administration)" endpoints.

So it may be better to clarify the potential issue in the readme 😉